a10_neutron_lbaas/plumbing/vthunder_per_tenant.py

```diff
@@ -44,12 +43,7 @@ def get_a10_client(self, device_info, **kwargs):
             return super(VThunderPerTenantPlumbingHooks, self).get_a10_client(device_info, **kwargs)
 
     def _instance_manager(self):
-        cfg = self.driver.config
-        vth = cfg.get_vthunder_config()
-        ks = a10_keystone.KeystoneA10(
-            cfg.get('keystone_version'), cfg.get('keystone_auth_url'), vth)
-        imgr = instance_manager.InstanceManager(ks_session=ks.session)
-        return imgr
+        return instance_manager.config_instance_manager(self.driver.config)
```

Comment:
Isn't this what the entry point used to look like?!? Or was that yesterday Cedric?